vars/waitForHelixRuns.groovy

```diff
@@ -148,5 +163,9 @@ def call (def helixRunsBlob, String prStatusPrefix) {
         timeout(720) {
             parallel helixRunTasks
         }
+        if(!passed) {
+            addSummaryLink('Failed Test Runs', failedRunMap)
```

Comment:
Can we update the html for the failures to highlight them in some way? Perhaps give them a red background or something?